flume/doc/design/architecture.md

```diff
@@ -1 +1,77 @@
-TBD
+#Cygnus architecture
```

Comment:
In that case, I'd suggest a "High-Availability Cygnus architecture" (in parallel to the "Basic Cygnus architecture") explaining that.